### PR TITLE
Fix mentions, roles and warns

### PR DESCRIPTION
**Pull request by stepech**
Some misc fixes
- add autoamtically added senator role under staff roles
- edit warns so that they don't reply to emhemeral message
- message welcoming user to new channel now also pings him

### CODE REVIEW

File: src/HonzaBotner/appsettings.CvutFit.json [hunk 1]
```diff
@@ -84,7 +84,9 @@
             "B-18928-SUMA-ZAMESTNANEC": [781137920278528012],
             "B-18923-SUMA-ZAMESTNANEC": [831601707955388456],
             "B-18000-SUMA-ZAMESTNANEC": [681574603759681578],
-            "B-18000-SUMA-PARTNER": [681574603759681578]
+            "B-18000-SUMA-PARTNER": [681574603759681578],
+            "B-00000-AS-STUDENT-FIT": [988749617611046952],
+            "B-18000-AS-STUDENTSKA-SEKCE-CLEN": [988749617611046952]
```
Comment (tenhobi) [inline src/HonzaBotner/appsettings.CvutFit.json:89]: Hmm, na to je role? :D

**Review by tenhobi — APPROVED** [review]

File: src/HonzaBotner/appsettings.CvutFit.json (hunk 1 above)
Comment (tenhobi) [reply src/HonzaBotner/appsettings.CvutFit.json:89]: A jen btw, není rovnou i role pro zaměstnance v AS? :)

Comment (stepech) [reply src/HonzaBotner/appsettings.CvutFit.json:89]: Když mluvíš o rolích, myslíš ty v senátu a nebo ty na discordu?

Comment (stepech) [reply src/HonzaBotner/appsettings.CvutFit.json:89]: A odkdy se tu nebavíme anglicky? 😃

Comment (stepech) [reply src/HonzaBotner/appsettings.CvutFit.json:89]: Ale ano, je i role pro zaměstnance. Tuším že se ale pár měsíců zpátky při zavádění discord role Academic senate rozhodlo o tom že jí zaměstnanci dostávat nebudou, protože na discordu jsou majorita studenti a lidi s tou rolí jsou tam hlavně pro konzultaci studentů a propagaci studentské agendy

Comment (tenhobi) [reply src/HonzaBotner/appsettings.CvutFit.json:89]: I don't know. I meant usermap roles for employees at AS FIT. `B-00000-AS-STUDENT-FIT` and `B-18000-AS-STUDENTSKA-SEKCE-CLEN` (what is the difference btw?) are for students at AS FIT, I suppose.

Comment (stepech) [reply src/HonzaBotner/appsettings.CvutFit.json:89]: B-00 is AS CVUT, B-18 is AS Fit, there are also staff roles indeed

Comment (tenhobi) [reply src/HonzaBotner/appsettings.CvutFit.json:89]: Btw. why don't we separate the "student" AS FIT and "student" AS CTU on Discord? 👀 Could you discuss that first, please? I think this separation would help students distinguish what topics each senator has knowledge of.

Comment (stepech) [reply src/HonzaBotner/appsettings.CvutFit.json:89]: Will be left like this for now. Might update in the future

**Pull request merged**